src/Graph.elm

```diff
@@ -1083,7 +1039,8 @@ ignorePath : SimpleNodeVisitor n e acc -> BfsNodeVisitor n e acc
 ignorePath visit path _ acc =
     case path of
         [] ->
-            Debug.crash "Graph.ignorePath: No algorithm should ever pass an empty path into this BfsNodeVisitor."
+            -- Debug.crash "Graph.ignorePath: No algorithm should ever pass an empty path into this BfsNodeVisitor."
```

Comment:
This will quietly fail.

Reply:
Again, I'd probably prefer an infinite loop here.

Reply:
Just create an auxiliary binding `crashLoop : String -> a; crashLoop reason = crashLoop reason`. It's crazy, but the only solution to this phony 'there are no crashes' madness.